tmt/cli.py

```diff
@@ -185,6 +185,10 @@ def get_command(  # type: ignore[override]
          multiple times.
          """)
 @verbosity_options
+@option(
+    '--log-time-on-terminal',
```

Comment:
Could possibly `--log-time` be enough? The current name sounds a little bit long to me...

Reply:
Time’s already logged, in the log file. Hence the “terminal”. I don’t mind changing the name, but to what?

Reply:
> Time’s already logged, in the log file.

Yeah, my point was that that time is logged in the log file, and always will be, so no future plans to have an option like `--log-time-in-the-log-file`, thus no need to differentiate, so `--log-time` would be just fine with me. Or perhaps something like `--show-time` or `--time-stamps`?

Reply:
`--show-time` sounds good, it bears the difference between "showing" logs on the terminal vs. "logging" them into a file.

Reply:
Renamed in e3031cb1d3320d6ef849ac72cb3f59a3cf954db5

Reply:
Good, thanks!